src/components/Dashboard/TodoList.vue

```diff
@@ -1385,8 +1386,9 @@ export default class TodoList extends Vue {
       case FilingTypes.CORRECTION:
         // see also FilingHistoryList.vue:correctThisFiling()
         switch (item.correctedFilingType) {
-          case FilingTypes.ALTERATION:
```

Comment:
This was the only case using FilingTypes instead of FilingNames leading to an bug where and 'alteration' would be checked against an 'Alteration'